src/rule.js

```diff
@@ -10,9 +10,11 @@
         email: /^([\w-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/,
         number: /^\d+$/,
         minlength: function(value, scope, element, attrs, param) {
+          value += '';
```

Comment:
The reason is as follows: if we use $modelValue whose value data type is Number, then value.length results in undefined; (curretlus used in the tests)

The question is, if the validator "maxlength" or "minlength" should be used with the Number data type; instead, maybe something like "maxVal" and "minVal" could be created on the app side. 

In that case, It would be probably desired to remove usage of the maxlength from tests.